app/learn/concepts/encryption-key/storj-vs-user-managed-encryption/page.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Storj Managed vs. User Managed Encryption
+docId: aitie6rohXai9uuv
+metadata:
+  title: Encryption passphrase management comparison
+  description:
+    Overview of the supported methods for managing an encryption passphrase
+    in  a Storj project, and their advantages/disadvantages.
+---
+
+When creating a new Storj project, users are able to select between "Storj Managed Encryption" and "Self Managed Encryption". Each has its own benefits and tradeoffs, and once an option is selected, it cannot be changed for that project.
+
+This page is intended to explain the difference between the two options, so that users can be better informed about which project encryption option is better suited to their use-case.
+
+## Brief Overivew
+
+**Storj Managed Encryption** is best suited to:
+* users who want a streamlined web browser experience, similar to other cloud storage providers
+* users who want to collaborate with others on the same project
+
+**Storj Managed Encryption** is best suited to:
+* users who want complete ownership of their data encryption keys
+* users who have more advanced/specialized encryption use-cases
+
+## More Details
```

Comment:
Let's get rid of the `## More details` and have the following sections just be two #'s.

 That'll render the table of contents better.


![Screenshot 2024-06-25 at 1 27 35 PM](https://github.com/storj/docs/assets/1147264/8858648c-b951-40aa-b4ff-4ccda67ebccd)